docs/design-decisions/ModelPromotion.md

```diff
@@ -0,0 +1,133 @@
+# Multi-Repository Model Development and Promotion Design
+
+## Overview
+
+This documentation outlines the design of an AI Model Factory consisting of two GitHub repositories: the Data Science / Prototyping Repository and the Model Factory / MLOps Repository. This design allows data scientists to focus on model development without the constraints of adhering to strict engineering practices, while ensuring that models are rigorously tested and deployed through automated processes.
+
+## What is a Model Factory?
+
+A Model factory based on DSToolkit v2 enables analytics and data science teams in partnership with software engineering teams to go from model development to a stable, reliable and auditable production model artifact built and maintained with strong software engineering fundamentals such as developer experience, testing, observability, release management, DevOps and automation.
```

Comment:
Model Factory is not "based on" DSToolkit. I don't know what does it mean.

Reply:
Updated, thanks!